__tests__/app.test.js

```diff
@@ -52,3 +51,64 @@ describe("GET api/categories", () => {
       });
   });
 });
+
+describe("GET api/reviews", () => {
+  test("should return an object with the key of reviews and an array of review objects as a value", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body: { reviews } }) => {
+        expect(reviews).toHaveLength(13);
+        reviews.forEach((review) => {
+          expect.objectContaining({
+            review_id: expect.any(Number),
+            title: expect.any(String),
+            category: expect.any(String),
+            designer: expect.any(String),
+            owner: expect.any(String),
+            comment_count: expect.any(Number),
+            review_img_url: expect.any(Number),
+            votes: expect.any(Number),
+          });
+        });
+      });
+  });
+});
+
+describe("GET api/reviews/:review_id", () => {
+  test("200: should return an object with the key of review and a review object as a value", () => {
+    return request(app)
+      .get("/api/reviews/3")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.review).toEqual({
+          review_id: 3,
+          title: "Ultimate Werewolf",
+          category: "social deduction",
+          designer: "Akihisa Okui",
+          owner: "bainesface",
+          review_body: "We couldn't find the werewolf!",
+          review_img_url:
+            "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+          created_at: "2021-01-18T10:01:41.251Z",
+          votes: 5,
+        });
+      });
```

Comment:
I'd like this happy path assertion to be more flexible as it currently is 'brittle', toMatchObject or expect.ObjectContaining are great matchers to keep the flexibility around so if more properties were added in future you'd be safe from failing this test.